Extend documentation (#12)
vkatsuba/rebar3_sheldon: README.md

[![Build Status][gh badge]][gh]
[![Erlang Versions][erlang version badge]][gh]

> A rebar plugin for spellchecking which is built based on [sheldon](https://github.com/inaka/sheldon).

Reviewer: ```suggestion
A rebar plugin for spellchecking code with [Sheldon](https://github.com/inaka/sheldon).
```

===> Youre welcome. And if he has twins, we can do all kinds of neat experiments on them.:
test/test_SUITE.erl:1: The word "Speling" in string is unknown. Maybe you wanted to use "speeling" or "speiling" or ....?
test/test_SUITE.erl:2: The word "fdfdf" in string is unknown.
test/test_SUITE.erl:3: The word "Unicode" in comment is unknown. Maybe you wanted to use "uncoded"?
```

## Command line
### Arguments List
```sh
$ rebar3 -h spellcheck
Plugin for spellcheck by sheldon
Usage: rebar3 spellcheck [-f <files>] [-i <ignore>] [-r <ignore_regex>]
                         [-d <default_dictionary>]
                         [-a <additional_dictionaries>]

  -f, --files                    List of files for spellchecker
  -i, --ignore                   List of ignore files for spellchecker
  -r, --ignore_regex             Regular exemptions for ignore lines
  -d, --default_dictionary       Set default dictionary
  -a, --additional_dictionaries  List of additional dictionaries
```
### Short full example
```sh
$ rebar3 spellcheck -f 'src/*.erl, test/*erl' -i 'include/*.hrl' -r '[_@./#&+-=*]' -d 'path/to/dict.txt' -a 'path/to.txt, additional_dict_1.txt'`
```

## Config
### Description

> By default is used dictionary of [sheldon](https://github.com/inaka/sheldon).

Reviewer: ```suggestion
By default, the dictionary used is the one provided by [sheldon](https://github.com/inaka/sheldon).
```

> If need use custom list of files for spellchecking - use config option `files`.

Reviewer: ```suggestion
If you need to use a custom list of files for spellchecking, use the config option `files`.
```

> If need ignore some files for spellchecking - use config option `ignore`.

Reviewer: ```suggestion
If you need to ignore some files for spellchecking, use the config option `ignore`.
```

> If need ignore some lines - use `ignore_regex` for set regular exemptions for ignore lines.

Reviewer: ```suggestion
If you need to ignore some particular strings or comments in your modules, use `ignore_regex` to set regular expressions and ignore the lines that match them.
```

> If `sheldon` dictionary is not suite at all - can be replaced by config option `default_dictionary`.

Reviewer: ```suggestion
If you want to directly replace Sheldon's dictionary with your own one, you can replace using the config option `default_dictionary`.
```

> If `sheldon` dictionary is suitable but should be expanded - use `additional_dictionaries` option to set own dictionaries for expand `sheldon` dictionary.

Reviewer: ```suggestion
If you need to _expand_ Sheldon's dictionary with your own words, use the `additional_dictionaries` option pointing to your own dictionaries with which you want to expand Sheldon's.
```

### Options

> Currently supported next options of `spellcheck` configuration:

Reviewer: ```suggestion
Currently supported options for `spellcheck` configuration through `rebar.config`:
```
